lib/cmd/compile/_client-init.js

```diff
@@ -50,3 +50,8 @@ function mountComponentModules() {
 // wait for DOMContentLoaded to initialize themselves, as the files themselves must be mounted first
 mountLegacyServices();
 mountComponentModules();
+window.process = window.process || {};
```

Comment:
this should probably happen _before_ those are called, right? in case their deps depend on it?

Reply:
Done!